bob-common/src/metrics/exporters/graphite_exporter/send.rs

```diff
@@ -179,12 +95,12 @@ async fn flush_gauges(
 }
 
 async fn flush_times(
-    times_map: &mut HashMap<MetricKey, TimeEntry>,
+    times_map: &HashMap<MetricKey, TimeEntry>,
     res_string: &mut String,
     prefix: &str,
     ts: i64,
 ) {
-    for (key, entry) in times_map.iter_mut() {
+    for (key, entry) in times_map.iter() {
         let mean_time = match entry.measurements_amount {
             0 => entry.mean.expect("No mean time provided"),
             val => entry.summary_time / val,
```

Comment:
I am not sure, but seems like we can take mean value here, because during snapshot creation mean value is calculated
UPD: it's [here](https://github.com/qoollo/bob/pull/295/files#diff-c891e00c677c35441c75d08c74b5df56bd0bfe50e426bb9dffaa2f2f50f33adcR44), but maybe it will be less obvious, so it would be better to leave a comment about this

Reply:
You're right, we already have mean value in entry. I don't think we need another comment for this, as "mean" is self explanatory